lstchain/image/muon/muon_analysis.py

```diff
@@ -291,3 +342,98 @@ def tag_pix_thr(image, thr_low = 50, thr_up = 500, pe_thr = 10):
     """
 
     return ((np.size(image[image > pe_thr]) < thr_up) and (np.size(image[image > pe_thr]) > thr_low))
+
+
+def radial_light_distribution(centre_x, centre_y, pixel_x, pixel_y, image):
```

Comment:
mixture of british and american english. code should be american english

Reply:
Done